org.kde.kcachegrind.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "org.kde.kcachegrind",
     "runtime": "org.kde.Platform",
-    "runtime-version": "5.15-21.08",
+    "runtime-version": "5.15-22.08",
```

Comment:
We're still testing this runtime: https://discourse.flathub.org/t/freedesktop-sdk-22-08-0-released/2871/3